packages/design-preview/src/components/style-variation.tsx

```diff
@@ -35,12 +36,12 @@ const StyleVariationPreview: React.FC< StyleVariationPreviewProps > = ( {
 	return (
 		<div
 			className={ classnames( 'design-preview__style-variation', {
-				'design-preview__style-variation--is-active': isActive,
+				'design-preview__style-variation--is-selected': isSelected,
 			} ) }
 			tabIndex={ 0 }
 			role="button"
-			onClick={ () => onClick?.( variation ) }
-			onKeyDown={ ( e ) => e.keyCode === SPACE_BAR_KEYCODE && onClick?.( variation ) }
+			onClick={ () => onClick( variation ) }
+			onKeyDown={ ( e ) => e.keyCode === SPACE_BAR_KEYCODE && onClick( variation ) }
```

Comment:
Same. It might be better to add `aria-label` here.

Reply:
Added `aria-label` "Style: ${ variation.title }`, but I wonder if the style titles are translated 🤔

Reply:
I checked the global styles in the editor and it won't be translated. You can see `aria-label="Pink"` in the screenshot below

![image](https://user-images.githubusercontent.com/13596067/189634604-05b28822-2228-46f8-aefb-4e727e5f1a0f.png)